code/game/machinery/atmos_field_gen.dm

```diff
@@ -0,0 +1,324 @@
+#define GENERATOR_INACTIVE 0 // off
+#define GENERATOR_WANTPOWER 1 // waiting for power/retrying, will activate and do things if powered
+#define GENERATOR_ACTIVE 2 // active
```

Comment:
Use autodocs, and you don't have to bother with unnecesary comments
![image](https://github.com/user-attachments/assets/41aa3442-65f1-4e06-b17a-e2e09e38b0bb)